src/moveFile.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+module.exports = function move(dir, file, data) {
+  fs.mkdir(dir, { recursive: true }, (e) => {
+    if (e) {
+    // eslint-disable-next-line no-console
+      console.log(e);
+
+      return;
+    }
+
+    fs.writeFile(path.join(dir, path.basename(file)), data, (err) => {
+      if (err) {
+      // eslint-disable-next-line no-console
+        console.log(e);
+
+        return;
+      }
+
+      fs.unlink(file, () => {});
```

Comment:
The unlink function should have an error handling callback, and it should only be called if the writeFile operation was successful.